src/app/features/admin-home/store/reducers.ts

```diff
@@ -0,0 +1,37 @@
+import { createReducer, on } from "@ngrx/store";
+import * as AdminHomeActions from "./actions";
+
+export interface AdminState {
+  probesInfo: {
+    version: string | null;
+    clusterEnabled: boolean | null;
+  };
+  error: any;
+}
+
+export const initialState: AdminState = {
+  probesInfo: {
+    version: null,
+    clusterEnabled: null,
+  },
+  error: null,
+};
+
+export const adminHomeReducer = createReducer(
+  initialState,
+  on(AdminHomeActions.fetchProbesInfo, (state) => ({
+    ...state,
+    error: null,
+  })),
+  on(AdminHomeActions.fetchProbesInfoSuccess, (state, { probesInfo }) => ({
+    ...state,
+    probesInfo: {
+      version: probesInfo.version,
```

Comment:
use ?. to check for null/undefined property value

Reply:
done